vllm_ascend/quantization/quantizer.py

```diff
@@ -54,3 +67,224 @@ def build_moe_method(self):
 
     def build_attention_method(self):
         raise NotImplementedError
+
+
+class MTAdaptedQuantizer:
```

Comment:
Emm, Looks like `MTAdaptedQuantizer` (MindTurboAdaptedQuantizer) is not a good name.

Reply:
I rename `MTAdaptedQuantizer` to `VLLMAscendQuantizer`